native/android/app/build.gradle

```diff
@@ -155,6 +155,7 @@ android {
 }
 
 dependencies {
+    implementation project(':@react-native-community_netinfo')
```

Comment:
Note: Starting from RN 60, these kind of line is not used anymore so this will need to be removed